kornia/color/gray_core.py

```diff
@@ -0,0 +1,206 @@
+from __future__ import annotations
+
+import jax.numpy as jnp
+import keras_core as keras
+import numpy as np
+import tensorflow as tf
+import torch
+
+
+def KORNIA_CHECK_IS_TENSOR(x: object, msg: str | None = None, raises: bool = True):
+    if not isinstance(x, [tf.Tensor, torch.Tensor, np.ndarray, jnp.array]):
+        if raises:
+            raise TypeError(f"Not a Tensor type. Got: {type(x)}.\n{msg}")
+        return False
+    return True
+
+
+def bgr_to_rgb(image):
+    if not isinstance(image, [tf.Tensor, torch.Tensor, np.ndarray, jnp.array]):
```

Comment:
Are you sure it is needed? I think that one of the benefits of keras.core is to avoid installation of the ALL frameworks, so we shouldn't import them all and check.

Reply:
Oh yeah. Again, just for the MVP. This would technically be handled by Keras-Core itself when needed

Reply:
More so, the image API actually. I imported them only for the `isinstance`-checks. That would probably end up going into the Image API anyway

Reply:
+1, the idea of using keras as multibackend handler, is to don't have any direct or indirect import/link to the backend itself in the codebase